drivers/SmartThings/sinux-britzyhub/profiles/vent.yml

```diff
@@ -0,0 +1,17 @@
+name: vent
+components:
+  - id: main
+    capabilities:
+      - id: switch
+        version: 1
+      - id: airPurifierFanMode
```

Comment:
We have a more generic `fanMode`  capability which I think may be more appropriate than this one which is intended to be specifically for air purifiers. Does the vent actually have a fan on it?